collegeElections/index.html

```diff
@@ -189,11 +139,8 @@
 
                  $http.jsonp(QueryCommand1).then(function(response) {
                     $scope.candidateReq.one = response.data
-                    //console.log('one');
                     $http.jsonp(QueryCommand2).then(function(response) {
                       $scope.candidateReq.two = response.data
```

Comment:
@kavithaenair my bad, there is no indentation problem here, the PR is good to go now.